src/pages/Scholarships.jsx

```diff
@@ -26,14 +29,55 @@ function ScholarshipsPage() {
   const [error, setError] = useState();
   const [sortField, setSortField] = useState('deadline');
   const [sortDir, setSortDir] = useState('asc');
+  const [amountFilterVals, setAmountFilterVals] = useState({
+    minAmount: 0,
+    maxAmount: 0,
+  });
```

Comment:
These don't need to be state variables as the URL parameter contains basically the properties of the page.